### PR TITLE
Fix issue when append_to_pre_release_tag is set

### PR DESCRIPTION
**Pull request by boxcee**
When `append_to_pre_release_tag` is set it should be used as identifier to look up the latest pre-release tag.

### CODE REVIEW

**Review by mathieudutour — APPROVED** [review]

**Comment by mathieudutour** [comment]
Thanks for your PRs! I don't have much time to maintain it and it looks like you are using it actively. Do you want me invite you as a collaborator to the repo? 

Collaborators are contributors who are given full write access; they can create branches on the repo, merge code, close issues, etc.

Here's what you need to know about becoming a collaborator:

	• You can work directly on the main repo, instead of using your own fork. Create branches for yourself and open pull requests from them. This makes it easier for other collaborators to quickly try out your changes locally, so it's encouraged. Please don't commit directly to master though :)
	• You can moderate issues, closing ones that are duplicates or not relevant, or reopening ones you feel deserve more discussion
	• You can submit code reviews for other folks' changes, accepting or requesting changes.
	• When reviewing small changes, feel free to merge it in when you feel the code is good to go. And for large, non-trivial changes, it might be good to get at least a couple approvals before landing (don't hesitate to ping me).

When it comes to your own pull requests, please don't accept and/or merge them; we should have at least 1 other pair of eyes on any changes before landing.

It's important to point out that there is no expectation of additional work. Feel free to accept the invitation even if you don't think you'll have much time to help contribute.

**Comment by boxcee** [comment]
Yes, I would like to help as collaborator.

**Pull request merged**
